topic: /velodyne_points
# topic: /livox/lidar
poseGT_path: /dataset/07.txt
pose_path: /dataset/07_result.txt
resolution: 0.4
iterCount_num: 15
nearestKSearch_num: 8
max_d: 0.1
show_goodFeatureNum_size: 1
cal_curva_rate: 4
show_pose_err: 0
